Formula/dsda-doom.rb

```diff
@@ -1,11 +1,27 @@
 class DsdaDoom < Formula
   desc "Fork of prboom+ with a focus on speedrunning"
   homepage "https://github.com/kraflab/dsda-doom"
-  url "https://github.com/kraflab/dsda-doom/archive/refs/tags/v0.24.3.tar.gz"
-  sha256 "d4cfc82eea029068329d6b6a2dcbe0b316b31a60af12e6dc5ad3e1d2c359d913"
   license "GPL-2.0-only"
+  revision 1
```

Comment:
These patches result in no user-facing changes, right? If so we can drop the `revision` to avoid users getting a no-op reinstall with `brew upgrade`.

Reply:
The patch that sets a custom location for `dsda-doom.wad` changes its install location to `libexec` and is no longer symlinked to `HOMEBREW_PREFIX/share/games/doom`. Additionally, the executable is compiled with `libexec` as an additional search path.

With this change, I'm not confident that the change is transparent to the user. But it's your call!

Reply:
Hm, ok. I guess we can keep it.